File: app/src/main/java/com/bogota/gdg/reactivexnetworkarchitecture/network/TeamsService.java
package com.bogota.gdg.reactivexnetworkarchitecture.network;

import com.bogota.gdg.reactivexnetworkarchitecture.json.Team;
import retrofit2.Call;
import retrofit2.http.GET;

import java.util.List;

/**
 * @author Santiago Carrillo.
 */

public interface TeamsService
{
    @GET( "teams.json" )
    Call<List<Team>> listTeams();
}

File: app/src/main/java/com/bogota/gdg/reactivexnetworkarchitecture/network/TeamsService.kt
package com.bogota.gdg.reactivexnetworkarchitecture.network

import com.bogota.gdg.reactivexnetworkarchitecture.json.Team
import retrofit2.Call
import retrofit2.http.GET

/**
 * @author Santiago Carrillo.
 */

interface TeamsService {
    @GET("teams.json")
    fun listTeams(): Call<List<Team>>
}
